rclcpp/src/rclcpp/list_parameter_override_prefixes.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 Intrisnic.ai
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <rclcpp/list_parameter_override_prefixes.hpp>
+
+#include <cassert>
+#include <string>
+#include <unordered_set>
+
+
+namespace rclcpp
+{
+std::unordered_set<std::string>
+list_parameter_override_prefixes(
+  node_interfaces::NodeInterfaces<node_interfaces::NodeParametersInterface> interfaces,
+  std::string prefix)
+{
+  const std::map<std::string, rclcpp::ParameterValue> & overrides =
+    interfaces.get_node_parameters_interface()->get_parameter_overrides();
+  return detail::list_parameter_override_prefixes(overrides, prefix);
+}
+
+std::unordered_set<std::string>
+detail::list_parameter_override_prefixes(
+  const std::map<std::string, rclcpp::ParameterValue> & overrides,
+  std::string prefix)
+{
+  // TODO(sloretz) ROS 2 must have this in a header somewhere, right?
```

Comment:
I am not sure if this is define in `rclcpp`. but there are lexer definition in `rcl`.

https://github.com/ros2/rcl/blob/60d80648c2a7cdbadcc7f22bdbc3c358c04dff7c/rcl/include/rcl/lexer.h#L79-L82